specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/SynapseLinkWorkspaces.json

```diff
@@ -0,0 +1,159 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/linkworkspaces": {
```

Comment:
>linkworkspaces [](http://example.com/codeflow?start=142&length=14)

looks like you only have the collection GET , do you not have GET operation for individual linkworkspaces resources ?

Reply:
Yes, we are just doing the collection for now.